homework_create_more_models/README.md

```diff
@@ -0,0 +1,269 @@
+# Homework: create comment model
+
+Now we only have Post model, what about receiving some feedback from your readers?
+
+## Creating comment blog model
+
+Let's open `blog/models.py` and append this piece of code to the end of file:
+
+    class Comment(models.Model):
+    	post = models.ForeignKey('blog.Post', related_name='comments')
+    	author = models.CharField(max_length=200)
+    	text = models.TextField()
+    	created_date = models.DateTimeField(default=timezone.now)
+    	approved = models.BooleanField(default=False)
+
+    	def approve(self):
+    		self.approved = True
+    		self.save()
+
+    	def __str__(self):
+    		return self.text
+
+You can go back to **Django models** chapter in tutorial if you need to remind yourself what each of field types means.
+
+In this chapter we have new type of field:
+- `models.BooleanField` - this is true/false field.
+
+And `related_name` option in `models.ForeignKey` allow us to have access to comments from post model.
+
+
+## Create tables for models in your database
+
+Now it's time to add our comment model to database. To do this we have to let know Django that we made changes in our model. Type `python manage.py makemigrations blog`. Just like this:
+
+    (myvenv) ~/djangogirls$ python manage.py makemigrations blog
+    Migrations for 'blog':
+      0002_comment.py:
+        - Create model Comment
+
+You can see that this command created for us another migration file in `blog/migrations/` directory. Now we need to apply those changes with `python manage.py migrate blog`. It should look like this:
+
+    (myvenv) ~/djangogirls$ python manage.py migrate blog
+    Operations to perform:
+      Apply all migrations: blog
+    Running migrations:
+      Rendering model states... DONE
+      Applying blog.0002_comment... OK
+
+Our Comment model exists in database now. It would be nice if we had access to it in our admin panel.
+
+## Register comment model in admin panel
+
+To register model in admin panel, go to `blog/admin.py` and add line:
+
+    admin.site.register(Comment)
+
+Don't forget to import Comment model, file should look like this:
+
+    from django.contrib import admin
+    from .models import Post, Comment
+
+    admin.site.register(Post)
+    admin.site.register(Comment)
+
+If you type `python manage.py runserver` in command prompt and go to [http://127.0.0.1:8000/admin/](http://127.0.0.1:8000/admin/) in your browser, you should have access to list, add and remove comments. Don't hesitate to play with it!
+
+## Make our comments visible
+
+Go to `blog/templated/blog/post_detail.html` file and add those lines before `{% endblock %}` tag:
+
+	<hr>
+	{% for comment in post.comments.all %}
+		<div class="comment">
+			<div class="date">{{ comment.created_date }}</div>
```

Comment:
Again different indentations in HTML. Here there are tabs, and below there are spaces :v: